package.json

```diff
@@ -129,6 +130,11 @@
     "test:php": "phpunit",
     "test:php:help": "npm run test:php -- --help"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
```

Comment:
Maybe this pre-commit hook is out of scope for this PR. If this is reverted, `package-lock.json` should also be reverted.

Reply:
Yes, let's omit that for now.

Reply:
Sure, I'll revert it

Reply:
Removed in https://github.com/ampproject/amp-wp/pull/4441/commits/740f3d130777dc07cda92ec464a762c81eb78890